components/mdc/Select/_index.scss

```diff
@@ -2,3 +2,7 @@
 @use '@material/menu-surface/mdc-menu-surface';
 @use '@material/menu/mdc-menu';
 @use '@material/select/styles';
+
+.mdc-floating-label--float-above {
```

Comment:
Why was this added?

Reply:
Floating labels never got the theme color. Here is the [issue](https://github.com/material-components/material-components-web/issues/2718). We have stopped using this label in cover, but I thought I'd fix it anyways.

Reply:
These also work ```
@use '@material/select' with (
  $focused-label-color: primary
)
```
The consumer can also include this in their _theme.scss file to choose secondary or whatever they want. 
or
```
/* MDC select floating label */
.mdc-select:not(.mdc-select--disabled).mdc-select--focused .mdc-floating-label{
  color: var(--mdc-theme-primary); /* or secondary */
}
```